frontend/test/storybook/visual-regression/v-filter-button.spec.ts

```diff
@@ -65,6 +59,7 @@ test.describe("VFilterButton", () => {
           pressed: true,
         })
         await page.locator("button", { hasText: "Filter" }).focus()
+        await sleep(500)
```

Comment:
```        
await sleep(500)
```

Could you add a comment explaining what this wait is for, @obulat? Otherwise it seems like something that could be easily accidentally deleted or forgotten. Perhaps a `todo: Remove when buttons are updated` or something as well if that's the case?

Reply:
I will open a PR based on `prepare_old_buttons` branch fixing the filter button focus ring today, so I think a todo comment is not necessary.